src/vscode-bicep/src/utils/appendToOutputChannel.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ext } from "../extensionVariables";
+
+export function appendToOutputChannel(text: string): void {
```

Comment:
Consider changing this to a `OutputChannelManager` class to encapsulate initializations and operations for all output channels.

Reply:
Refactored as part of this commit:
https://github.com/Azure/bicep/commit/62ec0b02cb7e0970c823c0b3f142ebe7fbc9b005

Thanks!